pkg/client/client_test.go

```diff
@@ -42,10 +42,19 @@ func TestNewClientOK(t *testing.T) {
 func TestNewClientFail(t *testing.T) {
 	// when
 	cl, err := client.NewClient("cool-token", "https://fail-cluster.com")
-
+	require.NoError(t, err)
+	assert.NotNil(t, cl)
 	// then
+	testObj := &toolchainv1alpha1.UserSignup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "john-doe",
+			Namespace: "default",
+		},
+	}
+	_, err = cl.IsObjectNamespaced(testObj)
```

Comment:
Let's add a comment explaining why we check if the object is namespaced (we just need to make ANY request to the actual API to trigger the error for an incorrectly configured client).

Reply:
done in [6bb562d](https://github.com/kubesaw/ksctl/pull/86/commits/6bb562dc8b23fb3b5a793191204662393a857418)